moped-editor/src/views/projects/projectsListView/useProjectListViewQuery/useProjectListViewQuery.js

```diff
@@ -3,41 +3,20 @@ import { gql } from "apollo-boost";
 
 export const useGetProjectListView = ({
   columnsToReturn,
-  exportColumnsToReturn,
   queryLimit,
   queryOffset,
   orderByColumn,
   orderByDirection,
   searchWhereString,
   advancedSearchWhereString,
+  queryName,
 }) => {
-  const { query, exportQuery } = useMemo(() => {
-    const queryString = `query ProjectListView {
-        project_list_view (
-            limit: ${queryLimit}
-            offset: ${queryOffset}
-            order_by: {${orderByColumn}: ${orderByDirection}}
-            where: { 
-              ${
-                searchWhereString && advancedSearchWhereString
-                  ? `_and: [{_or: [${searchWhereString}]}, {${advancedSearchWhereString}}]`
-                  : ""
-              }
-              ${
-                searchWhereString && !advancedSearchWhereString
-                  ? `_or: [${searchWhereString}]`
-                  : ""
-              }
-              ${
-                advancedSearchWhereString && !searchWhereString
-                  ? advancedSearchWhereString
-                  : ""
-              }
-            }
-        ) {
-            ${columnsToReturn.join("\n")}
-        },
-        project_list_view_aggregate (
+  const { query } = useMemo(() => {
```

Comment:
This works, but the destructuring could go along with the returned object from `useMemo` now that it is only returning one variable.